### PR TITLE
FEAT:ONEPLAT-1063-E2E tests: Write test cases for Developer console.

### PR DESCRIPTION
**Pull request by supriyagawade**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix
 Write test cases for Developer console.

## Does this PR introduce a breaking change
no

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: e2e/integration/devconsoletests.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return false;
+    } );
+
+    it( 'Test for create the application', () => {
+        cy.contains( 'Create new App', { timeout: 5000 } ).click()
+        cy.get( '#app-name' ).type( 'e2e test automation' )
+        cy.get( '#app-path' ).type( 'e2e test automation' )
+        cy.get( '#app-desc' ).type( 'e2e test automation' )
+        cy.contains( 'Create App' ).click()
+        cy.contains( 'App Created Succssfully!' ).should( 'be.visible' );
```
Comment (riginoommen) [inline e2e/integration/devconsoletests.js:18]:
```suggestion
        cy.contains( 'App Created Successfully!' ).should( 'be.visible' );
```

File: e2e/integration/devconsoletests.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    } );
+
+    it( 'Test for delete the application', () => {
+       cy.contains('App Settings').click()
+       cy.contains('Delete this App').click()
+       cy.get( '#delete-app', { timeout: 5000 } ).type( 'e2e test automation' )
+       cy.contains( 'I understand the consequences, delete this app' ).click()
+       cy.contains( 'App Deleted Succssfully!' ).should( 'be.visible' )
```
Comment (riginoommen) [inline e2e/integration/devconsoletests.js:26]:
```suggestion
       cy.contains( 'App Deleted Successfully!' ).should( 'be.visible' )
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: e2e/integration/devconsoletests.js (hunk 1 above)
Comment (supriyagawade) [reply e2e/integration/devconsoletests.js:18]: @riginoommen We need to spell it correctly in the application pop-up, I used the same for assertion!

File: e2e/integration/devconsoletests.js (hunk 2 above)
Comment (supriyagawade) [reply e2e/integration/devconsoletests.js:26]: We need to spell it correctly in the application pop-up, I used the same for assertion!

**Review by riginoommen — APPROVED** [review]

File: e2e/integration/devconsoletests.js (hunk 1 above)
Comment (ghanlohar) [reply e2e/integration/devconsoletests.js:18]: @supriyagawade Let's put the correct word in the test cases. we can ask @deshmukhmayur to update the message.

**Review by ghanlohar — APPROVED** [review]

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**